dist/leaflet.distortableimage.js

```diff
@@ -1129,7 +1129,7 @@ L.LockHandle = L.EditHandle.extend({
neO9e6w76NvWxjHXUCepzlBC/xDAdzOj3nmaaODYxo4KYeUUrPRJr0maxBHJ0CRnS5yCmO8vyNthlSUXsLLSz6nOe1yNxp7mI/T/DnvGJaQsmch8C0HQqZ3hU1IkW1UY0aTRq0mOEOfp3b59x3WXezlzlcBiwwzylOcJSXOZXru5hhPdM0qeOkpt6lHzC1JlYfMgtz+wzp0+YiZ3O+HbaROWapE7DICS4kb2HMCf6IAyzQoU2HrmlGJAmeUrtVESlM66JoNEWLWe7gn3Gz47yPd7GXFj3mOckJXuM1XqOHWBts1tHCY8h8Otb5zPNH/B2XWKJNmy5do5LxW6ndatkURaSwNrqZf8TNOHdyO5vpcYJjHOEIxxhpLK8pPheSWJT2axO/QJcDBOZmQtQGL+i88eK2v9M3EjfyC+zHsbiZFuc4yfMc4qxG8WRfCjfySc5yFN/cph3HMhWwTipuREqmjja5l/fgwg7qnOYkz4TPIolJvxTu5AP8MaP4Fq3LGLuYT9W6hZTIiuORh8csH6SFBR7nOMKzN37TVawODvfxt3QYxHeUHMbF7NuVISLt5NawRbbASxzV8CzSWZzhPo7Sp2cedhlhF7NKsgurkR1P1H8r06FI89KoeOPvTqbjZ6FcM/nYkkirmdw5RqVbKrWES/laDpvMHMkajrkpV8B1mYosUhSTdmg8Fpg6W3DxzGPu2dUdJNIKa5SOSR4zWtW70A2HbWbacbT8SiHPZpEjUhSVmhqNhU7uNpnlUZy4QlJEWtWoZMezG0SRRZqNFwdQarcm79wyy5w4Go2FpmkiUYHjUZFTu+QriVTsiNQw1W52/VqJtKo6WUUtTkWMl1oBqrBns8hrVidtB1FknEwUUmq3BhpFXykmlUEkilshFbdGKkt6KhJ9rKucX4m0JlqJoibphc4vdDUXk1PtFviSKJGEsgqJJCSRRBKiNEgkISSSUHInkfQJhEQSQkgkISSSEBJJGbWQSEKI8omk2CQkkhASSQghkYSQSEJIJCEkkhBCIgkhkYSQSEJIJCGERBJCIgkhkYSQSEIIiSSERBJCIgkhkYQQEkkIiSSERBJCIgkhJJIQEkkIiSSEkEhCSCQhJJIQEkkIIZGEkEhCSCQhJJIQQiIJIZGEkEhCSCQhhEQSQiIJIZGEkEhCCIkkhEQSQiIJIZGEEBJJCIkkhEQSQiIJISSSEBJJCIkkhJBIQkgkISSSEBJJCCGRhJBIQkgkISSSEEIiCSGRhJBIQkgkIYREEkIiCSGRhJBIQgiJJIREEkIiCSGRhBASSQiJJIREEkJIJCEkkhASSQiJJISQSEJIJCEkkhASSQghkYSQSEJIJCEkkhBCIgkhkYSQSEJIJCGERBJCIgkhkYSQSEIIiSSERBJCIgkhkYQQEkkIiSSERBJCSCQhJJIQEkkIiSSEkEhCSCQhJJIQEkkIIZGEkEhCSCQhJJIQQiIJIZG+H4FOoJBIQkikiYlDvk6ikEhCSCQhVOtKJCEkklA8kkhCKLXTaRBCEUnoQiiRrngadCdJIkkknYbK40ukSVFJMhVdpEAirXUUChhrLBaacXw5DCTSWkolkYrNIHVxDIoanezC6hPEXw01FgvN0JzNoMh1b9EjUiCRCn8GO3Fa5xc3vbMLLlHAmK5GY6FZMAIFy3INibSqiV3AosZigfE5T0CAb16KSKt+AgLz64La34UW6Qw+Y8ZxXCpk/664zQbMdWzMa5rbUGD6zMfRyC9ulVTc1C4wh93nhNoNBa5zz7NoIlI6JkmkVYtG4fVrzJjXWFByV1iRXqJnNBrjxzJJpFWLSKFGI0YscVgiFZQehxkyYsSIMWNT9yq1W9XULryKDenxbbpSqZDn8TinzeVwZM6nUrtVPQFhYjdiyIghL/GUGg6FbDQ8SpchQxOVxnEDXF27VVMpikcDBrT5GqelUsHwOcwRBvFryEgRafUTu8CkBAP69DnOl7ik9K5QZ/EM36BDnz79WCNfNdJaRKQRQ/r06NHlab4slQp0/i7yF5wxGoUqRcldIc+hW9gTEUakATX69PCocYAhH2aL1qEoQFI3z9c4Qte8evQZMiyuRkUWKWp/D+hRw8XF5iku8X5uo46l0TqxZ27IKzzC63TiV5cegzgeFbJGcgt6MiwCfEbYOLi4ONhYWLzCF7iVd3MTDWzpNIESneVxnqVNjw7tWKR+SiRFpFVPEHxGDHCwjUbh3x3kBbZyM7ewhSYejvk3JNaqixOl4T4jFjnBsxw12nTp0GGRtolHQ0ZFfh7JKt77tqLfLBwcang0aDJFy/xqUKdOjRo1POo0aVKnTh3bxC6k1gpKE87MDyf8DBnRp2/St2gOQ5++EalNmzY9czcpNUGoaOPSLfSp840KVtx+CO8sRSIlsepycaTQyiqV/jpcJ2gcV7WJSF069OgzYlTkp5GKLFJUJ6XXoAkb4mmRwtrJjnVLYpFYeZ2C1LkJjEZDhvTp04v7ddn6SDXSGp2scSo+RXeW6tTw4iaEHUek9C+xOiIlazEkc1EilfoMTDQaFVujoqd2llEpe8Xr45mGeDoiWXE0spTcrUpql6gUPcscTU8dmilBg9Qcu4Iv81n0iGSZK15yZ2lIjX5Ko3SNZCkerUn7IVJpbOajjMx04/RE1ULHoyJ37bJ/tLCxsXFwcIxEDpZpfX+/xE5q5dlguFJyF8UkP5YpepQvaTEE3+8bSqSVFIlYDyulk21e1jKRLv/fJdHKyZRePTW9UpCfUugqsUgirYVI6Y6cZWqitERZhdQCXx2ZltdKPum1n4LMWnZIpMkQiUzEscDcdr28LpI8a9d4CFIVLVeXSCKtrUhXizdv9DdidSPTD7zCt2Y2TM4pDKTNxDckSoOlJ+GEuHH0EJwQEkkIiSSERBJCSCQhJJIQEkkIiSSEkEhCSCQhJJIQEkkIIZGEkEhCSCQhJJIQQiIJIZGEkEhCSCQhhEQSYk34/55dj6jwQIHlAAAAAElFTkSuQmCC',
```

Comment:
That is one huge URL! Should we put it in another file ar a variable elsewhere?

Reply:
Reverted back to Xs: https://github.com/publiclab/Leaflet.DistortableImage/pull/358#discussion_r305636537